packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -784,30 +784,19 @@ -(NSDictionary *)getUpdatesConfig: (nullable NSDictionary *) constants
     runtimeVersion = _updatesInterface.runtimeVersion ?: @"";
   }
 
-  // the project url field is added to app.json.updates when running `eas update:configure`
-  // the `u.expo.dev` determines that it is the modern manifest protocol
-  NSString *projectUrl = @"";
-  if (_updatesInterface) {
-    projectUrl = [constants valueForKeyPath:@"manifest.updates.url"];
-  }
-
-  NSURL *url = [NSURL URLWithString:projectUrl];
+  NSString *expoUpdatesURLString = [_updatesInterface.updateURL absoluteString] ?: @"";
```

Comment:
I'm not sure if we can rely on this given that it will change in dev-launcher interface every time the user launches a different update

Reply:
Ah, I see. Okay, yep, then I think the way you have it structured is fine.

I think if I were writing this function from scratch I'd probably just use a regex that tests for `[staging-]?u.expo.dev/(uuid)` since that is the project URL, but I agree that dev client is a bit wonky since it changes the update URL when launching updates.